SampleProject/VOKCoreDataManagerTests/ManagedObjectImportExportTests.m

```diff
@@ -452,7 +452,8 @@ - (void)testFetchWithURI
     person = nil;
     [[[VOKCoreDataManager sharedInstance] managedObjectContext] reset];
 
-    VIPerson *personFromURI = [[VOKCoreDataManager sharedInstance] existingObjectAtURI:uri forManagedObjectContext:nil];
+    VIPerson *personFromURI = (VIPerson *)[[VOKCoreDataManager sharedInstance] existingObjectAtURI:uri
```

Comment:
If this still needs the cast, then I don't see the point in having `__kindof`, but I may well be missing the point (I kind of feel like that's likely... pun maybe intended).